modules/salt/pages/formula-yomi.adoc

```diff
@@ -0,0 +1,153 @@
+[[yomi-formula]]
+= Yomi Formula
+
+Yhe Yomi (yet one more installer) installer for {suse} and openSUSE operating systems is configured using formulas with forms.
+
+The ``yomi-formula`` package provides these formulas:
+
+* Yomi
+* Yomi Storage
+* Yomi Bootloader
+* Yomi Software
+* Yomi Services
+* Yomi Users
+
+
+.Procedure: Install the Yomi Formulas with Forms
+. On the {productname} Server, at the command prompt, as root, install the ``yomi-formula`` package:
++
+----
+zypper in yomi-formula
+----
+. In the {susemgr} {webui}, open the details page for the Salt master, and navigate to the Formulas tab.
```

Comment:
```suggestion
. In the {susemgr} {webui}, open the details page for the Salt master, and navigate to the Formulas tab.
```
In the suma context, this is at least confusing.  The Salt master runs on the suma server.  Is this a second Salt master?  Needs explanation.  Is this supported?

Reply:
You have to go into some system details page to be able to enable the formulas, so it seemed sensible to make it the salt master. I've changed it to "any Salt client", maybe that's less confusing?

Reply:
I've removed this bit, in light of the next comment :arrow_down: